### PR TITLE
Add support for the multiple attribute

### LINKED ISSUE #2: Add support for multiple files
If you set the `multiple` attribute, you can select several files. Right now, the first file is always chosen, and there's no way to get to the remaining ones. I fixed it in my local fork of this, but only for the `onChange` event.

I'd love to create a pull request that does this for all, but then I need to know that this is actually maintained.

### PR DESCRIPTION
**Pull request by selbekk**
If you need to upload several files at once, this patch lets you do that.

Fixes #2

### CODE REVIEW

File: src/FileInput.js [hunk 1]
```diff
@@ -54,46 +54,50 @@ const FileInput = React.createClass({
 
   handleChange: function(event){
     const {readAs, cancelIf, onCancel, onProgress, abortIf, onChange} = this.props;
-    const file = event.target.files[0];
+    const files = event.target.files
```
Comment (greena13) [inline src/FileInput.js]: The readme should be updated to document the new behaviour for multiple files.

File: src/FileInput.js [hunk 2]
```diff
@@ -54,46 +54,50 @@ const FileInput = React.createClass({
...
+              if(abortIf(event, file)){
+                fileReader.abort();
+              } else if(onProgress){
+                onProgress(event, file);
+              }
+            }
+          } else if(onProgress) {
+            fileReader.onprogress = onProgress;
+          }
+          if(multiple) {
```
Comment (greena13) [inline src/FileInput.js:96]: I'm sorry, I've searched this file but I can't find where `multiple` is actually defined. Is there something obvious I am missing?

Comment (selbekk) [inline src/FileInput.js:96]: `multiple` is passed as a prop by the user, so it isn't specified in the code anywhere. What this commit actually does is to make the code agnostic about how many files the user selects.

File: src/FileInput.js [hunk 3]
```diff
@@ -54,46 +54,50 @@ const FileInput = React.createClass({
...
+            file.forEach()
```
Comment (greena13) [inline src/FileInput.js:97]: What is occurring here when `forEach` is called without any arguments?

File: src/FileInput.js (hunk 2 above)
Comment (greena13) [inline src/FileInput.js:96]: Thanks for explaining. I am a little confused about how `multiple` is in scope here and why `this.props.multiple` is not required.

**Comment by selbekk** [comment]
This looked a bit shifty I must admit. Closed the PR for now, perhaps I'll create a new one in the future :)

**Pull request closed without merging**